src/components/LaunchDetailsPage/components/LaunchCardContent/components/CardDetails/CardDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Typography } from '@mui/material';
+import { launchDetails } from '../../../../mocks/launchDetails';
+import './CardDetails.scss';
+
+export const CardDetails = () => {
+  const detailsCaptions: string[] = ['Area:', 'Status:', 'Date:'];
+  const detailsValues: string[] = [
+    `${launchDetails.pad.name}, ${launchDetails.pad.location.name}`,
+    launchDetails.status.name,
+    launchDetails.net,
```

Comment:
all items should be in a column

Reply:
row 9 contains one item which consist of two values